src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -67,12 +69,23 @@ public EiffelPluginConfiguration(final String pluginName, final NameKey project,
         }
         // Read plugin configuration
         this.enabled = pluginConfig.getBoolean(ENABLED, false);
-        this.filter = pluginConfig.getString(FILTER);
+        if (pluginConfig.getStringList(FILTER)!=null && pluginConfig.getStringList(FILTER).length > 0) {
```

Comment:
Please move the filter string list to a local variable so you only need to use .getStringList once. Once this is done you may use StringUtils.isEmpty(filterList.toString()) StringUtils checks both fotr null and empty.

Reply:
Fixed in https://github.com/eiffel-community/eiffel-gerrit-plugin/pull/35/commits/22af4864a1073d3c6e87e840dd6338d9ea7757b1